src/app/providers/slot/slot.actions.ts

```diff
@@ -0,0 +1,11 @@
+import { Action } from '@ngrx/store';
+
+export const SLOT_HAS_CHANGED = '[SlotProvider] SLOT_HAS_CHANGED';
+
+export class SlotHasChanged implements Action {
```

Comment:
These actions will also need to be updated but can do it at a later time (in my story perhaps)